src/worldfactory.cpp

```diff
@@ -1414,6 +1414,27 @@ void load_world_option( JsonObject &jo )
     }
 }
 
+void load_external_option( JsonObject &jo )
+{
+    auto name = jo.get_string( "name" );
+    auto stype = jo.get_string( "stype" );
+    if( !get_options().has_option( name ) ) {
```

Comment:
Would be much more readable if you would use a local reference variable instead of calling `get_options()` over and over again (`options_manager &opts = get_options(); opts.add(...)`).